apps/src/p5lab/poetry/constants.js

```diff
@@ -28,10 +28,10 @@ export const PALETTES = {
 
 // Notes:
 // - author is not translated.
-// - Poems are shown in all languages, unless there is a locale attribute, in which case
-//   the poem is shown in the dropdown only for users with that current locale.
-// - If the locale attribute is set, then title is used and is not translated.
-// - If the locale attribute is set, then linesSplit is used and is not translated.
+// - Poems are shown in all languages, unless there is a locales attribute, in which case
+//   the poem is shown in the dropdown only for users with a current locale that matches.
```

Comment:
super nit: "that matches" is slightly ambiguous, perhaps "that is included in the array"?